[BUG]Mysql pod keep restarting for OOM killed on VM

kbcli version
Kubernetes: v1.25.10+rke2r1
KubeBlocks: 0.5.3-beta.2
kbcli: 0.5.3-beta.2

This only failed on vm, does not failed on cloud

```
kbcli cluster create mysqltest2              --termination-policy=Delete              --monitor=false --enable-all-logs=false --cluster-definition=apecloud-mysql --set cpu=100m,memory=0.5Gi,replicas=1,storage=1Gi  --namespace default

root@yjtest0:/home/mjingz# k get pod | grep mysqltest2
mysqltest2-mysql-0                                      3/4     Error     1 (4m16s ago)   5m53s

Log when pod is killed
2023-06-21 04:07:10+00:00 [Note] [KubeBlocks]: Temporary server started.
'/var/lib/mysql/mysql.sock' -> '/var/run/mysqld/mysqld.sock'
ERROR 2013 (HY000) at line 1: Lost connection to MySQL server during query
/apecloud/mysql/scripts/libmysql.sh: line 394:   472 Killed                  "$@" --skip-networking --default-time-zone=SYSTEM --socket="${SOCKET}"

root@yjtest0:/home/mjingz# k logs mysqltest2-mysql-0
Defaulted container "mysql" out of: mysql, metrics, kb-checkrole, config-manager
2023-06-21 04:34:50+00:00 [Note] [MySQL]: Entrypoint script for MySQL Server  started.
2023-06-21 04:34:50+00:00 [Note] [MySQL]: KB_EMBEDDED_WESQL=1
2023-06-21 04:34:50+00:00 [Note] [KubeBlocks]:
2023-06-21 04:34:50+00:00 [Note] [KubeBlocks]: Welcome to the KubeBlocks MySQL Cluster Container
2023-06-21 04:34:50+00:00 [Note] [KubeBlocks]:
2023-06-21 04:34:50+00:00 [Note] [KubeBlocks]: KB_VOLUME_DIR=/data/mysql
2023-06-21 04:34:50+00:00 [Note] [KubeBlocks]: KB_DATA_DIR=/data/mysql/data
2023-06-21 04:34:50+00:00 [Note] [KubeBlocks]: KB_LOG_DIR=/data/mysql/log
2023-06-21 04:34:50+00:00 [Note] [KubeBlocks]: KB_CLUSTER_LEADER=
2023-06-21 04:34:50+00:00 [Note] [KubeBlocks]: KB_CLUSTER_FOLLOWERS=
2023-06-21 04:34:50+00:00 [Note] [KubeBlocks]: KB_CLUSTER_UID=53cd783f-98dd-4793-a387-b7cbbccc713d
2023-06-21 04:34:50+00:00 [Note] [KubeBlocks]: KB_CLUSTER_UID_PATH=/data/mysql/.kb_cluster_uid
2023-06-21 04:34:50+00:00 [Note] [KubeBlocks]: KB_MYSQL_CONF_FILE=/opt/mysql/my.cnf
2023-06-21 04:34:50+00:00 [Note] [KubeBlocks]: KB_MYSQL_PORT=3306
2023-06-21 04:34:50+00:00 [Note] [KubeBlocks]: KB_POD_NAME=mysqltest2-mysql-0
2023-06-21 04:34:50+00:00 [Note] [KubeBlocks]: KB_POD_N=1
2023-06-21 04:34:50+00:00 [Note] [KubeBlocks]: KB_POD_INDEX=0
2023-06-21 04:34:50+00:00 [Note] [KubeBlocks]: KB_POD_HOST=mysqltest2-mysql-0.mysqltest2-mysql-headless
2023-06-21 04:34:51+00:00 [Note] [KubeBlocks]:
2023-06-21 04:34:51+00:00 [Note] [KubeBlocks]:
2023-06-21 04:34:51+00:00 [Note] [KubeBlocks]: CLUSTER_INFO=mysqltest2-mysql-0.mysqltest2-mysql-headless:13306@1
2023-06-21 04:34:51+00:00 [Note] [KubeBlocks]: CLUSTER_LEADER_HOST=
2023-06-21 04:34:51+00:00 [Note] [KubeBlocks]:
2023-06-21 04:34:51+00:00 [Note] [KubeBlocks]: MYSQL_EXTRA_CONF_FILE=/opt/mysql/my.cnf
2023-06-21 04:34:51+00:00 [Note] [KubeBlocks]:
2023-06-21 04:34:51+00:00 [Note] [KubeBlocks]: Injecting configuration /opt/mysql/my.cnf to /etc/my.cnf
2023-06-21 04:34:51+00:00 [Note] [KubeBlocks]: Injecting configuration /etc/mysql/wesql_cluster.cnf to /etc/my.cnf
2023-06-21 04:34:52+00:00 [Note] [KubeBlocks]: DATADIR=/data/mysql/data/
2023-06-21 04:34:52+00:00 [Note] [KubeBlocks]: PID_FILE=/var/run/mysqld/mysqld.pid
2023-06-21 04:34:53+00:00 [Note] [KubeBlocks]: SOCKET=/var/run/mysqld/mysqld.sock
2023-06-21 04:34:55+00:00 [Note] [KubeBlocks]: Switching to dedicated user 'mysql'
2023-06-21 04:34:55+00:00 [Note] [MySQL]: Entrypoint script for MySQL Server  started.
2023-06-21 04:34:55+00:00 [Note] [MySQL]: KB_EMBEDDED_WESQL=1
2023-06-21 04:34:56+00:00 [Note] [KubeBlocks]: DATADIR=/data/mysql/data/
2023-06-21 04:34:57+00:00 [Note] [KubeBlocks]: PID_FILE=/var/run/mysqld/mysqld.pid
2023-06-21 04:34:57+00:00 [Note] [KubeBlocks]: SOCKET=/var/run/mysqld/mysqld.sock
2023-06-21 04:34:59+00:00 [Note] [KubeBlocks]: JEMALLOC_ENABLED=1
2023-06-21 04:34:59+00:00 [Note] [KubeBlocks]: Using jemalloc: LD_PRELOAD=/usr/local/lib/libjemalloc.so.2
2023-06-21 04:35:00+00:00 [Note] [KubeBlocks]: Using persisted data
2023-06-21 04:35:00+00:00 [Note] [KubeBlocks]: Starting temporary server in background: 'mysqld --consensus-enabled=0 --disable-log-bin'
2023-06-21 04:35:00+00:00 [Note] [KubeBlocks]: Waiting for server startup
2023-06-21 04:35:00+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:02+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:04+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:05+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:07+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:08+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:10+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:11+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:12+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:13+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:15+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:16+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:17+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:19+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:20+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:21+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:22+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:24+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:25+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:26+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:27+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:29+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:30+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:31+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:33+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:34+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:35+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:36+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:38+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:39+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:40+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:41+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:42+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:44+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:45+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:46+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:48+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:49+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:50+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:51+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:53+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:54+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:55+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:56+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:58+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:35:59+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:00+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:01+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:03+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:04+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:05+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:07+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:08+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:09+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:11+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:12+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:13+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:14+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:16+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:17+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:18+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:19+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:20+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:22+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:23+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:24+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:25+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:27+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:28+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:29+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:30+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:32+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:33+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:34+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:35+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:37+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:38+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:39+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:41+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:42+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:43+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:44+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:45+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:47+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:48+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:49+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:51+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:52+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:53+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:54+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:56+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:57+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:36:58+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:00+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:01+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:02+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:03+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:04+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:06+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:07+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:08+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:09+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:10+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:12+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:13+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:14+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:15+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:17+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:18+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:19+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:21+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:22+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:23+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:24+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:26+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:27+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:28+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:30+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:31+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:32+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:34+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:35+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:36+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:37+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:39+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:40+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:41+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:42+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:44+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:45+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:46+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:47+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:48+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:50+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:51+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:52+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:54+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:55+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:56+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:57+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:37:59+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:00+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:01+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:03+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:04+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:05+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:06+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:08+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:09+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:10+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:12+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:13+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:14+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:15+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:17+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:18+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:19+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:20+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:22+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:23+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:24+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:25+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:27+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:28+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:29+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:30+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:32+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:33+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:34+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:35+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:37+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:38+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:39+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:40+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:42+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:43+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:44+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:46+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:47+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:48+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:50+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-06-21 04:38:51+00:00 [ERROR] [KubeBlocks]: Unable to start server.
2023-06-21 04:38:51+00:00 [Note] [KubeBlocks]: Stopping temporary server(pid: 6656).
mysqladmin: connect to server at 'localhost' failed
error: 'Access denied for user 'root'@'localhost' (using password: YES)'
2023-06-21 04:38:51+00:00 [ERROR] [KubeBlocks]: Unable to shut down server.

root@yjtest0:/home/mjingz# kbcli cluster describe mysqltest2
Name: mysqltest2	 Created Time: Jun 21,2023 04:24 UTC+0000
NAMESPACE   CLUSTER-DEFINITION   VERSION           STATUS   TERMINATION-POLICY
default     apecloud-mysql       ac-mysql-8.0.30   Failed   Delete

Endpoints:
COMPONENT   MODE        INTERNAL                                          EXTERNAL
mysql       ReadWrite   mysqltest2-mysql.default.svc.cluster.local:3306   <none>

Topology:
COMPONENT   INSTANCE             ROLE     STATUS    AZ       NODE                    CREATED-TIME
mysql       mysqltest2-mysql-0   <none>   Running   <none>   yjtest3/10.128.15.203   Jun 21,2023 04:24 UTC+0000

Resources Allocation:
COMPONENT   DEDICATED   CPU(REQUEST/LIMIT)   MEMORY(REQUEST/LIMIT)   STORAGE-SIZE   STORAGE-CLASS
mysql       false       100m / 100m          512Mi / 512Mi           data:1Gi       local-path

Images:
COMPONENT   TYPE    IMAGE
mysql       mysql   registry.cn-hangzhou.aliyuncs.com/apecloud/apecloud-mysql-server:8.0.30-5.alpha9.20230606.gf80d546.9

Data Protection:
AUTO-BACKUP   BACKUP-SCHEDULE   TYPE     BACKUP-TTL   LAST-SCHEDULE   RECOVERABLE-TIME
Disabled      <none>            <none>   7d           <none>          <none>

Show cluster events: kbcli cluster list-events -n default mysqltest2

```

--- a/deploy/apecloud-mysql/config/mysql8-config.tpl
+++ b/deploy/apecloud-mysql/config/mysql8-config.tpl
@@ -32,7 +32,12 @@ innodb_buffer_pool_size={{ $pool_buffer_size }}
 max_connections={{ div ( div $phy_memory 4 ) $single_thread_memory }}
 {{- end}}
 
-# alias replica_exec_mode. Aliyn slave_exec_mode=STRICT
+# if memory less than 8Gi, disable performance_schema
+{{- if lt $phy_memory 8589934592 }}
+performance_schema=OFF
+{{- end }}
+
+# alias replica_exec_mode. Aliyun slave_exec_mode=STRICT
 slave_exec_mode=IDEMPOTENT
 
 # gtid

